packages/docgen/src/markdown/formatter.js

```diff
@@ -100,7 +100,7 @@ module.exports = function( rootDir, docPath, symbols, headingTitle, headingStart
 			formatTag(
 				'Parameters',
 				getSymbolTagsByName( symbol, 'param' ),
-				( tag ) => `\n- *${ tag.name }* \`${ tag.type }\`: ${ cleanSpaces( tag.description ) }`,
+				( tag ) => `\n- *${ tag.name }*${ tag.type ? ` \`${ tag.type }\`` : '' }: ${ cleanSpaces( tag.description ) }`,
```

Comment:
How about explicitly state that the type is `unknown`? I think it provides more clarity to the reader of the API than nothing at all.

Reply:
It's possible, yeah. I wasn't sure how best to call it in a way where it wouldn't be misinterpreted as the type (even "Unknown" could mislead one into reading it as some form of type). I'm on the fence whether it's more clear one way than the other, especially in mind of this.

Reply:
I'm not attached particularly to `unknown`, perhaps `UnknownType` is a better choice? Whatever works is fine for me. My point was more about this being like any other situation: we want to gracefully degrade and communicate the user what's happening.

Reply:
I added "(unknown type)".

I'm hoping it's only needed as an interim solution anyways, 'til we can improve types parsing here.